src/senaite/core/browser/samples/view.py

```diff
@@ -143,22 +143,9 @@ def __init__(self, context, request):
                 "toggle": True}),
             ("ClientID", {
                 "title": _("Client ID"),
-                "index": "getClientID",
```

Comment:
Why did you remove this Index? Without this it is no longer possible to sort the columns by client ID.

Reply:
Thanks for the heads-up. Restored with https://github.com/senaite/senaite.core/pull/2335/commits/1b6e61d23174eaa1730538a481d06852ad4d03a6